GameServerLib/Players/PlayerManager.cs

```diff
@@ -101,5 +101,17 @@ public ClientInfo GetClientInfoByChampion(IChampion champ)
 
             return _players;
         }
+
+        public ClientInfo GetClientInfoWithPlayerId(int playerId)
```

Comment:
Remove this function and use `GetPeerInfo`.

Reply:
Oh i didnt notice there was a Function that did this already